std/src/main/scala/cats/std/map.scala

```diff
@@ -0,0 +1,38 @@
+package cats
+package std
+
+
+trait MapInstances {
+  type MapWithK[A] = Map[A, _]
```

Comment:
thank for all the hints, I'm kind of new to the implementation of all of this concept, I'll try to pursue the implementation later in the day